config/bidderinfo.go

```diff
@@ -243,10 +252,49 @@ func processBidderInfos(reader InfoReader, normalizeBidderName func(string) (ope
 				return nil, fmt.Errorf("error parsing config for bidder %s: %v", fileName, err)
 			}
 
-			infos[string(normalizedBidderName)] = info
+			bidderInfos[string(normalizedBidderName)] = info
+
+			//need to maintain nullable fields from BidderInfo struct into bidderInfoNullableFields
+			//to handle the default values in aliases yaml
+			if len(info.AliasOf) > 0 {
+				aliasInfo := bidderInfoNullableFields{}
+				if err := yaml.Unmarshal(data, &aliasInfo); err != nil {
+					return nil, fmt.Errorf("error parsing config for aliased bidder %s: %v", fileName, err)
+				}
+
+				aliasInfos[string(normalizedBidderName)] = aliasInfo
+			}
 		}
 	}
-	return infos, nil
+	return processBidderAliases(aliasInfos, bidderInfos)
+}
+
+func processBidderAliases(aliasInfos aliasInfos, bidderInfos BidderInfos) (BidderInfos, error) {
+	for bidderName := range aliasInfos {
+		if err := validateAliases(bidderInfos[bidderName], bidderInfos, bidderName); err != nil {
+			return nil, err
+		}
+		aliasBidderInfo := bidderInfos[bidderName]
```

Comment:
Here, we lookup twice on `bidderInfos` for same bidder name on line 274 and 275. Also look  up on `bidderInfos` map is done using  key from `aliasInfos` map. So should check `aliasBidderInfo` is present before making call to `validateAliases()`




```
aliasBidderInfo, ok := bidderInfos[bidderName]
if !ok {
    // return error 
}

if err := validateAliases(aliasBidderInfo, bidderInfos, bidderName); err != nil {
    return nil, err
}
...
```

Reply:
https://github.com/prebid/prebid-server/pull/3000/commits/92211a53da0a3a1dcf80c64c792f36265d88ffcb